en/concept/custom_mixer_payload.md

```diff
@@ -0,0 +1,77 @@
+
+For this example, we'll create a custom mixer based on the *RC passthrough mixer* ([pass.aux.mix](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/mixers/pass.aux.mix)).
+This mixer is commonly loaded into the AUX PWM ports on large multicopters). 
+It passes through the values of 4 user-defined RC channels (set using the [RC_MAP_AUXx/RC_MAP_FLAPS](../advanced/parameter_reference.md#RC_MAP_AUX1) parameters) to the first four outputs on the AUX PWM output.
+
+```
+# Manual pass through mixer for servo outputs 1-4
+
+# AUX1 channel (select RC channel with RC_MAP_AUX1 param)
+M: 1
+S: 3 5  10000  10000      0 -10000  10000
+
+# AUX2 channel (select RC channel with RC_MAP_AUX2 param)
+M: 1
+S: 3 6  10000  10000      0 -10000  10000
+
+# AUX3 channel (select RC channel with RC_MAP_AUX3 param)
+M: 1
+S: 3 7  10000  10000      0 -10000  10000
+
+# FLAPS channel (select RC channel with RC_MAP_FLAPS param)
+M: 1
+S: 3 4  10000  10000      0 -10000  10000
+```
+
+> **Note** The file defines a [simple mixer](../concept/mixing.md#simple-mixer) with four outputs.
+> - `M: 1` indicates an output that is defined by one control input (the following `S` line).
+> - `S: 3 `_`n`_ indicates that the input is the n<>th<> input of [Control Group 3 (Manual Passthrough)](../concept/mixing.md#control-group-3-manual-passthrough).
+>  So for `S: 3 5` the input is called "RC aux1", and will output the value of the channel mapped by parameter `RC_MAP_AUX1`.
+> - The order of output declaration of 
+
+
+Start by copying the mixer file and putting it onto the SD Card at: **/etc/mixers/pass.aux.mix** (see [Mixing and Actuators > Loading a Custom Mixer](../concept/mixing.md#loading_custom_mixer).
+
+Remove the first section with a payload control group function input:
+- Change this:
+  ```
+  # AUX1 channel (control group 3, RC CH5) (select RC channel with RC_MAP_AUX1 param)
+  M: 1
+  S: 3 5  10000  10000      0 -10000  10000
+  ```
+- To:
+  ```
+  # Payload 1 (control group 6) channel 1
+  M: 1
+  S: 6 1  10000  10000      0 -10000  10000
+  ```
+
+Because this output is in the first position in the file it will map to the first AUX PWM output (unless UAVCAN is enabled).
+This output will now respect updates to the payload control group (6) output 1.
+
+Putting an output on group 6 works by publishing actuator control group 6. First you have to create the publication.
+This should happen once when the PX4 module is initialized (look for places where this pattern is already being used):
+```
+uORB::Publication<actuator_controls_s> _actuator_controls_pub{ORB_ID(actuator_controls_0)};
+```
+
+Then you need to publish the first message:
+```
+actuator_controls_s _act_controls{};
+_act_controls.timestamp = hrt_absolute_time();
+// set the first output to 50% positive (this would rotate a servo halfway into one of its directions)
+_act_controls.control[0] = 0.5f;
```

Comment:
And further to the comment above - https://github.com/PX4/Devguide/pull/973/files#r392069246 is this a typo too? I still don't know how we know we have the right control group, but again would have thought that the index would be 1 not 0.

Reply:
Looks like both are 0-based indexes, so you are right, the index should match with the mixer definition.

Reply:
@bkueng But (even) if I change this to `_act_controls.control[1] = 0.5f;` and we publish the topic for group 6 (as discussed above https://github.com/PX4/Devguide/pull/973/files#r392069246) how does the system know that I want to publish to group 6? ie.. `actuator_controls_s _act_controls{};` would be *all groups*, and we would want to publish to group 6, index 1.

Does my question make sense?

Reply:
The topic name defines the group: `uORB::Publication<actuator_controls_s> _actuator_controls_pub{ORB_ID(actuator_controls_0)};` is group #0, `uORB::Publication<actuator_controls_s> _actuator_controls_pub{ORB_ID(actuator_controls_1)};` is group #1. Does this make sense?

Reply:
I think my C++ is a bit rusty. I can see 
```
#publish to group 6
uORB::Publication<actuator_controls_s> _actuator_controls_pub{ORB_ID(actuator_controls_6)};
```
But I can't see how you know that this now means `_act_controls` now points to the same set of controls (actuator_controls_6)
```
actuator_controls_s _act_controls{};
```
Anyway, can I ask YOU to fix up this whole section https://github.com/PX4/Devguide/pull/973/files#r392899091 - ie Putting an output on group 6 works

Too much back and forth otherwise?

Reply:
`_act_controls` is a generic data structure. It can be published on any control group topic.
I pushed a commit to add it.

Reply:
Thanks!

Reply:
Thanks. I'll just have to trust this works.